bounties/other/adevait/hrm/2/README.md

```diff
@@ -0,0 +1,14 @@
+# Description
+
+`adevait/hrm` is vulnerable to `Stored Cross-Site Scripting (XSS)`
+
+https://github.com/adevait/hrm
+
+# Steps To Reproduce-:  
+
+1) install https://github.com/adevait/hrm
+2) login as admin
+3) while adding any events the titile is vulnerable 
```

Comment:
Make more detailed the description (note it's not a `event` to be created but a `company`)